justfile

```diff
@@ -200,6 +200,10 @@ init:
 down *flags:
     just dc down {{ flags }}
 
+# Take all services down then call the specified app's up recipe. ex.: `just dup catalog` is useful for restarting the catalog with new environment variables
```

Comment:
A shorter comment would help as this is printed as the help output. The example doesn't add much and can be left out, if that helps.

Reply:
Also what does "dup" mean, is is an abbreviation for something? I don't get it.